services/address-lookup/src/utils/capitalizeString.js

```diff
@@ -0,0 +1,15 @@
+module.exports =
+  /**
+   * Capitalize uppercase string
+   *
+   * @param {string} uppercaseString
+   * @returns {string}
+   */
+  (uppercaseString) =>
```

Comment:
Why do we need this level of complexity rather than just doing String.ToUpperCase()?? I'm not clear what this is for/does

Reply:
Ignore the above, i just had a look at the below! :D